zanata-war/src/main/java/org/zanata/security/permission/PermissionEvaluator.java

```diff
@@ -41,43 +42,31 @@
 @ApplicationScoped
 public class PermissionEvaluator {
 
-    private static final String ALL_ACTION_GRANTER = "__**__";
+    public static final String ALL_ACTION_GRANTER = "__**__";
```

Comment:
Could we move `PermissionProvider` into this package and make this package-private?